lib/asset.js

```diff
@@ -2,8 +2,8 @@ const { Asset } = require('parcel-bundler');
 const svg = require("svgo");
 
 class InlineSvgAsset extends Asset {
-  constructor(name, pkg, options) {
-    super(name, pkg, options);
+  constructor(name, options) {
+    super(name, options);
```

Comment:
Could just `super.apply(this, arguments)` or `super(...arguments)` to future-proof this.

Reply:
yeah , can do this, better change `arguments` to  other name like `args`,  arguments is a keyword.

Reply:
In this case, the use of the keyword is actually intended; we want to pass precisely the same arguments that were passed to us, to `super()`, so we do that with `super(...arguments)` (or the ES5 equivalent).